packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -237,7 +237,7 @@ module.exports = {
       .findOne({ where: { email: email.toLowerCase() } });
 
     if (!user || user.blocked) {
-      return ctx.send({ ok: true });
+      return ctx.throw(404, 'User not found or access blocked');
```

Comment:
Hi, this is intentional to avoid brute force attacks being able to test for all existing users in the DB.